packages/common/src/store/buy-usdc/types.ts

```diff
@@ -0,0 +1,20 @@
+export enum USDCOnRampProvider {
+  COINBASE = 'coinbase',
+  STRIPE = 'stripe',
+  UNKNOWN = 'unknown'
+}
+
+export type PurchaseInfo = {
+  /**
+   * Desired amount of USDC in *cents*
```

Comment:
How do you two feel about the mix of cents and USDC Wei @dharit-tan / @schottra ? Would be easy to change the backend back to the "Wei everywhere" approach if this complicates things

Reply:
Personally in favor of wei everywhere, in case we decide to handle more granular splits.

Reply:
I liked the consumer-friendly nature of the price in a track metadata being in a slightly more readable format. And also that it makes it easier to cap things to cent boundaries. I would be fine to go back to everything wei all the time. I'd just like us to make up our minds 😁 

A couple of other thoughts:
* If this saga accepted a value in wei, we could make sure we only buy exactly the amount you need to cover the purchase (supposing you had fractional cents in your wallet or something like that)
* The Stripe modal wants it desired amount in **dollars** as a floating point number, which adds another conversion point to try and remember.

Reply:
Yeah if it's working we don't have to change anything! Just taking a temperature if it's hard to remember and juggle these conversions.

From your thoughts, it really sounds like we should have _dollars_ in the API results or WEI, but not cents?

Reply:
Eh, I think floating point dollar values open us up to the same problems with prices that we had with Wei: When writing metadata, you can provide a price that's not on a cent boundary.

I think we can leave this for now. We may need to revisit it